tests/src/helpers.ts

```diff
@@ -20,11 +21,11 @@ const animalPoll: Poll = {
 	}],
 };
 
-export async function createUser(): Promise<User> {
+export async function createUser(appId?: string): Promise<User> {
 	const uniqueId = generateId();
 	const user = await (User.new({
 		appUserId: `test_${uniqueId}`,
-		appId: "kik",
+		appId: appId || (await Application.findOne())!.id,
```

Comment:
what, a random application?

Reply:
I'm with Nitzi. I'm also not sure the `await` is needed there.

Reply:
The await is needed. I just want a user attached to some application so it's a valid user. not all tests care about which application the user belongs to, as long as it belongs to one.